### ISSUE TITLE
[QUESTION] - Repository blacklist

### ISSUE THREAD

**Issue by alexandresavicki** [SvanBoxel/organization-workflows#25]
There's a way to exclude a list of repositories to not run workflow?

I running workflow for an organization that's have some git ops repositories, this repositories are receiving push commits in to deffault branch at any time by some automation, so these repositories doesn't need run workflow.

I'm running my workflow with configuration bellow:

```
on:
  push:
    branches: [ $default-branch ]
  pull_request:
    branches: [ $default-branch ]
```
So i try to setup to run only on `pull_request` :

```
on:
  pull_request:
    branches: [ $default-branch ]
```
But doesn't work, it's still running for push events for any branch.

My main problem with this is that can exceed my Action runs quota easily.

**Comment by SvanBoxel** [comment]
Hi @alexandresavicki! 

Yes, this is one of the config options I want to add to the [app configuration](https://github.com/SvanBoxel/organization-workflows#app-configuration).

Basically an include/exclude option that accepts an array of that repository names. Would that fit your use case?

**Comment by SvanBoxel** [comment]
This functionality is introduced in PR #26 and released with release [1.2.0](https://github.com/SvanBoxel/organization-workflows/releases/tag/1.2.0) of this app. 🎉

**Comment by alexandresavicki** [comment]
Hi @SvanBoxel Thanks a lot for this feature.

I was testing here, i don't know if's i'm doing something wrong, buts isn't working as expected.

I've created `organization-workflows-settings.yml` in my .github repository:

![image](https://user-images.githubusercontent.com/8751353/105759538-5808b780-5f2f-11eb-81d7-e73183bc4906.png)


The propertie `include_workflows_repository` work's from me.

So I pushed some commits in my repository `example-app-savicki` and it keeps running.
![image](https://user-images.githubusercontent.com/8751353/105759666-740c5900-5f2f-11eb-8da8-c4011013236f.png)


Any idea what's i'm doing wrong?
